package.json

```diff
@@ -1,5 +1,9 @@
 {
 	"private": true,
+    "scripts": {
+        "start": "npx mix watch",
+        "build": "npx mix --production"
+    },
```

Comment:
Just added `npm start` as a shortcut for mix watch. I'm lazy.

Reply:
sounds good - we can make make this standard in other places too